mod_ci/controllers.py

```diff
@@ -903,6 +903,63 @@ def show_maintenance():
     }
 
 
+@mod_ci.route('/blocked_users')
+@login_required
+@check_access_rights([Role.admin])
+@template_renderer()
+def blocked_users():
+    return {
+        'blocked_users': BlockedUsers.query.order_by(BlockedUsers.userID)
+    }
+
+
+@mod_ci.route('/blocked_users')
```

Comment:
Multiple definitions for a single route? You don't need to write function defs here.

`@mod_ci.route('/blocked_users')` for this you'll write exactly what should happen if someone opens this path. Check other snippets in the same file to get an idea.

Reply:
Done, correctly I think 🤦‍♂️